static/vendor/javascripts/polyfillio.fallback.js

```diff
@@ -0,0 +1,5482 @@
+ * - String.prototype.contains, License: CC0 (required by "_ArrayIterator", "Array.prototype.@@iterator", "es6", "Array.prototype.entries", "Array.prototype.keys")
+ * - Symbol.toStringTag, License: MIT (required by "es6", "_ArrayIterator", "Array.prototype.@@iterator", "Array.prototype.entries", "Array.prototype.keys")
+ * - _ArrayIterator, License: MIT (required by "Array.prototype.@@iterator", "es6", "Array.prototype.entries", "Array.prototype.keys")
+ * - Array.prototype.@@iterator, License: CC0 (required by "es6", "Array.prototype.values")
+ * - Array.prototype.entries, License: CC0 (required by "es6")
+ * - Array.prototype.find, License: CC0 (required by "es6")
+ * - Array.prototype.findIndex, License: CC0 (required by "es6")
+ * - Array.prototype.keys, License: CC0 (required by "es6")
+ * - Array.prototype.values, License: MIT (required by "es6")
+ * - Function.name, License: MIT (required by "es6")
+ * - Math.acosh, License: CC0 (required by "es6")
+ * - Math.asinh, License: CC0 (required by "es6")
+ * - Math.atanh, License: CC0 (required by "es6")
+ * - Math.cbrt, License: CC0 (required by "es6")
+ * - Math.clz32, License: CC0 (required by "es6")
+ * - Math.cosh, License: CC0 (required by "es6")
+ * - Math.expm1, License: CC0 (required by "es6")
+ * - Math.hypot, License: CC0 (required by "es6")
+ * - Math.imul, License: CC0 (required by "es6")
+ * - Math.log10, License: CC0 (required by "es6")
+ * - Math.log1p, License: CC0 (required by "es6")
+ * - Math.log2, License: CC0 (required by "es6")
+ * - Math.sign, License: CC0 (required by "es6")
+ * - Math.sinh, License: CC0 (required by "es6")
+ * - Math.tanh, License: CC0 (required by "es6")
+ * - Math.trunc, License: CC0 (required by "es6")
+ * - Number.MAX_SAFE_INTEGER, License: MIT (required by "es6")
+ * - Number.MIN_SAFE_INTEGER, License: MIT (required by "es6")
+ * - Number.isFinite, License: MIT (required by "es6")
+ * - Number.isInteger, License: MIT (required by "es6")
+ * - Number.parseFloat, License: MIT (required by "es6")
+ * - Number.parseInt, License: MIT (required by "es6")
+ * - Object.is, License: CC0 (required by "es6")
+ * - String.prototype.repeat, License: CC0 (required by "es6")
+ * - Symbol.hasInstance, License: MIT (required by "es6")
+ * - Symbol.isConcatSpreadable, License: MIT (required by "es6")
+ * - Symbol.match, License: MIT (required by "es6")
+ * - Symbol.replace, License: MIT (required by "es6")
+ * - Symbol.search, License: MIT (required by "es6")
+ * - Symbol.split, License: MIT (required by "es6")
+ * - Symbol.toPrimitive, License: MIT (required by "es6")
+ * - Symbol.unscopables, License: MIT (required by "es6")
+ * - WeakMap, License: https://github.com/webcomponents/webcomponentsjs/blob/master/LICENSE.md (required by "es6")
+ * - WeakSet, License: https://github.com/webcomponents/webcomponentsjs/blob/master/LICENSE.md (required by "es6")
+ * - Array.prototype.includes, License: CC0 (required by "es7") */
+
+(function(undefined) {
```

Comment:
it's minified during build, left it like this so we can see what's in it if we need to